packages/components/src/molecules/SearchInputField/SearchInputField.tsx

```diff
@@ -88,6 +88,7 @@ const SearchInputField = forwardRef<
         {...otherProps}
       />
       <IconButton
+        data-fs-icon-button="null"
```

Comment:
Do we really need this? 👀

Reply:
Yes 😞, without this the Search icon doesn't appear when we are searching I on mobile.

Without
![Screenshot 2023-04-05 at 13 20 35](https://user-images.githubusercontent.com/51174217/230142678-dff48ae9-1d81-4326-b0b3-619462a166a1.png)

With
![Screenshot 2023-04-05 at 13 21 10](https://user-images.githubusercontent.com/51174217/230142827-cd370894-a618-4c4a-9aed-89af7ffd9eac.png)

Reply:
oh, I see.